### ISSUE TITLE
Show additional tooltip for block that under scaffolds

### ISSUE THREAD

**Issue by Makishima** [WarpzoneStudios/Hammerting#84]
![изображение](https://user-images.githubusercontent.com/5548003/110013758-168bd900-7d33-11eb-9bec-04dde2162e44.png)

**Comment by Srekel** [comment]
Do you mean we should show both tooltips?

**Comment by Makishima** [comment]
Yes. If we can see a background tile through the building there should be some kind of both tooltips.
Especially it is useful in non-lighted areas where it is hard to say is it iron or copper.

**Comment by Makishima** [comment]
Or you can combine it as for water
![изображение](https://user-images.githubusercontent.com/5548003/110108849-ba21cb80-7dbd-11eb-973a-deb45ba861e0.png)
